### PR TITLE
speedtest-go: add new package

### PR DESCRIPTION
**Pull request by teleostnacl**
This is a Command Line Interface (CLI) and pure Go API to test internet speed using speedtest.net. Its upstream is https://github.com/showwin/speedtest-go

Maintainer: TeleostNaCl [teleostnacl@gmail.com](mailto:teleostnacl@gmail.com)
Compile tested: MediaTek Ralink MIPS MT7621 based boards Phicomm K2P Powered by LuCI Master (git-24.096.15376-b029f06)
Run tested: MediaTek Ralink MIPS MT7621 based boards Phicomm K2P Powered by LuCI Master (git-24.096.15376-b029f06)

Description: Add new package, a CLI with go to test internet speed

### CODE REVIEW

File: net/speedtest-go/Makefile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+PKG_NAME:=speedtest-go
+PKG_VERSION:=1.6.10
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/showwin/speedtest-go/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=3f2d6ae3b60d8b6e972038fd2e13f3fdeca75647d88785d65bfa0d99a2f4fd79
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
```
Comment (1715173329) [inline net/speedtest-go/Makefile:16]:
```suggestion
PKG_LICENSE_FILES:=LICENSE
PKG_MAINTAINER:=TeleostNaCl Dai <teleostnacl@gmail.com>
```

as the PR description please add yourself as the maintainer
or you can assign me

**Comment by 1715173329** [comment]
1. v1.6.11 is out, please bump to that version
2. please squash your commits

**Review by 1715173329 — APPROVED** [review]

**Comment by 1715173329** [comment]
Merged, thank you!

**Comment by teleostnacl** [comment]
> Merged, thank you!

thanks very much

**Pull request merged**